docs/source/1.0/guides/building-models/build-config.rst

```diff
@@ -287,6 +287,47 @@ the :ref:`tags trait <tags-trait>`.
     This transformer does not remove shapes from the prelude.
 
 
+.. _excludeShapesByTrait-transform:
+
+excludeShapesByTrait
+--------------------
+
+Removes shapes if they are marked with a specific trait.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 10 20 70
+
+    * - Property
+      - Type
+      - Description
+    * - trait
+      - ``string``
```

Comment:
Should this be a list of traits instead? All the `ByTag` transformers take lists, for reference.

Reply:
Done